tests/ts-test/tsconfig.json

```diff
@@ -0,0 +1,45 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "es2018",
+    "module": "esNext",
+    "lib": ["esnext", "dom"],
+    "allowJs": true,
+    "checkJs": false,
+    "jsx": "preserve",
+    "incremental": true,
+    "noEmit": true,
+
+    /* Strict Type-Checking Options */
+    "strict": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+
+    /* Additional Checks */
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": false,
+    "noFallthroughCasesInSwitch": false,
+
+    /* Module Resolution Options */
+    "baseUrl": ".",
+    "paths": {
+      /* Support absolute /imports/* with a leading '/' */
+      "/*": ["*"],
+      /* Workaround for https://github.com/zodern/meteor-types/issues/10 */
+      "meteor/react-meteor-data": [
+        "./node_modules/@types/meteor/react-meteor-data.d.ts"
+      ],
+      "meteor/*": ["./.meteor/local/types/packages.d.ts"]
```

Comment:
the typescript skeleton is broken - it should fetch meteor types from zodern:types but instead fetches from the old DefinitelyTyped files in node_modules